src/app/AttributePathParams.h

```diff
@@ -23,46 +23,27 @@
 
 namespace chip {
 namespace app {
-enum class AttributePathFlags : uint8_t
+enum class AttributePathSelectorFlag : uint8_t
 {
+    kInValid        = 0x00,
```

Comment:
do we need the kInvalid flag?

BitFlags mValue is specifically initialized to 0. I think that flags should actually be 'bits' and not include a constant for "nothing set"